redis-server.py

```diff
@@ -1,23 +1,150 @@
 # redis-server.py
 
 import socket
+import argparse
+import threading
+import time
 
 from utils import server_response_decode
 
 HOST = "127.0.0.1"  # Standard loopback interface address (localhost)
-PORT = 65430  # Port to listen on (non-privileged ports are > 1023)
-MY_DICT = {}
+PORT = 6380  # Port to listen on (non-privileged ports are > 1023)
+DATA_DICT = {}  # Dictionary for storage of data on "Redis" server
+TIME_DICT = {}  # Dictionary for storing key and time (value) for SET time based options
+BUFFER_SIZE = 1024
+SEPARATOR = "\r\n"
```

Comment:
Rather than define this in each file, it could live in `utils` and be imported.

Reply:
moved